examples/text_labels_and_annotations/mathtext_fontfamily_example.py

```diff
@@ -0,0 +1,38 @@
+"""
+========================
+Math Fontfamily Example
+========================
+
+A simple example showcasing the new *math_fontfamily* parameter that can
+be used to change the family of fonts for each individual text
+element in a plot. If no parameter is set, the global value
+:rc:`mathtext.fontset` will be used.
+"""
+
+import matplotlib.pyplot as plt
+
+plt.figure(figsize=(9, 4))
+
+# A simple plot for the background.
+plt.plot(range(11), color="0.9", label=r'Math Font Example')
+
+# Here we hold the text, we will mix normal text and math text
+msg = r"This is a normal text and "
+msg+= r"$this\ is\ a\ text\ in\ math\ mode,\ look,\ I\ can\ do\ this\ "
+msg+= r"\int_{0}^{\infty } x^2 dx !$"
```

Comment:
Use implicit string concatenation:
```suggestion
msg = (r"This is a normal text and "
       r"$this\ is\ a\ text\ in\ math\ mode,\ look,\ I\ can\ do\ this\ "
       r"\int_{0}^{\infty } x^2 dx !$")
```